deku/deku.d.ts

```diff
@@ -0,0 +1,137 @@
+// Type definitions for deku v2.0.0-rc16
+// Project: https://github.com/anthonyshort/deku
+// Definitions by: Sho Fuji <https://github.com/pocka/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = deku;
+export as namespace deku;
+
+declare namespace deku {
+
+	interface VirtualElement {
+		type: string;
+	}
+
+	/**
+	 * Create a DOM renderer using a container element.
+	 * Everything will be rendered inside of that container.
+	 * Returns a function that accepts new state that can replace what is currently rendered.
+	 */
+	function createApp(el: HTMLElement): Render;
```

Comment:
This can be handled using an optional argument: `dispatch?: Function`. Also, can you provide a more specific type than `Function`?